magento/adobe-stock-integration
# 326: add permanent error message on invalid API key

### Description (*)
According to the issue description, the error message that signalised about incorrect connection has to be displayed permanently.

I decided to implement this in the following way: at the block which renders data for the test connection button template, the connection is tested and result provided to the js plugin. If the connection failed, the error message shown.

The validate-api-key js code also refactored.

### Fixed Issues (if relevant)
#326 

### Manual testing scenarios (*)
1. Set incorrect API key
2. Save setting page
3. Check that the error message appears and can not be closed.
